tests/__init__.py

```diff
@@ -213,11 +214,21 @@ def nocompat_test_suite():
     return result
 
 
+@patch("dulwich.protocol.DEFAULT_GIT_PROTOCOL_VERSION_FETCH", new=0)
```

Comment:
I guess this works and is okay for now but ideally this would just be a parameter on the test classes.

Reply:
Please check the new version, it should be better.